src/aker_metrics.h

```diff
@@ -119,4 +110,12 @@ int get_blocked_mac_count(const char* blocked);
 /* Destroys the metrics resoureces. */
 void destroy_akermetrics();
 
+
+/**
+ *  Calculates and returns the timezone offset value set in current process
+ *
+ *  @return the timezone offset value in secs (eg.for +0600 the offset return is +14400)
+ *
+ */
+long int get_tz_offset();
```

Comment:
This isn't needed outside aker_metrics.c.  It would be better to not include this here & make the function static with the comment in the c file here:
https://github.com/xmidt-org/aker/pull/186/files#diff-cc816a2b62a2405e5545808cd733ce674f4fae8fc65a882b7d84fbd59722e40aR296

Reply:
Pushed the recent changes with static function , please have a look